packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/queue/lambda/index.py

```diff
@@ -0,0 +1,16 @@
+import os
+import boto3
+from queue import QueueHandler
+
+
+def queue_handler(event, context):
+    """
+    Handler for the lambda trigger
+    """
+
+    ecs = boto3.client('ecs')
+    sqs = boto3.client('sqs')
```

Comment:
To continue the conversation:
> It's done for CDK as well! But our function implementation is done in QueueHandler.emit() and the clients are defined as globals for it so I think we are still following this practice.

They don't seem to be globals, they're instance variables afaict. Right now, the clients will be initialized on each invocation of the Lambda fn. Whereas if they were globals, it'd improve the execution time slightly.

This Lambda fn is not performance critical, so it doesn't really matter and we can keep it as is 👍 . But I just wanted to clarify the difference 📚

Reply:
Ahh right, our Lambda calls the `queue_handler()` so the clients aren't global still. Thanks for pointing out :thumbsup: